opendevin/core/config.py

```diff
@@ -331,8 +331,8 @@ def load_from_toml(config: AppConfig, toml_file: str = 'config.toml'):
     try:
         with open(toml_file, 'r', encoding='utf-8') as toml_contents:
             toml_config = toml.load(toml_contents)
-    except FileNotFoundError:
-        # the file is optional, we don't need to do anything
+    except FileNotFoundError as e:
+        logger.warning(f'Config file not found: {e}')
```

Comment:
No I think this should be a `info`, not a warning. Warning means the user has done something wrong. By design, this config file is optional.